src/main/java/teammates/logic/automated/FeedbackSubmissionAdjustmentAction.java

```diff
@@ -81,13 +81,13 @@ public boolean execute() {
                 try {
                     stLogic.adjustFeedbackResponseForEnrollments(enrollmentList, response);
                 } catch (Exception e) {
-                    log.severe(String.format(errorString, sessionName, courseId, e.getMessage()));
+                    log.severe(String.format(errorString, sessionName, courseId, e.getMessage(), e.getStackTrace().toString()));
```

Comment:
I remember seeing a utility method somewhere that extracts the stack trace as a string.